src/pages/en/installation-new.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Installation
+description: How to install Astro with NPM, PNPM, or Yarn.
+---
+Use npm, pnpm or yarn to create and set up a new Astro project locally!
+
+
+## Prerequisites
+
+- **Node.js** - `14.15.0`, `v16.0.0`, or higher.
+- **Text editor** - We recommend [VS Code](https://code.visualstudio.com/) with our [Official Astro extension](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode).
+- **Terminal** - Astro is accessed through its command-line interface (CLI).
+
+## 1. Install Astro
+
+Run one of the following commands in your terminal to start our handy install wizard, `create-astro`. This will walk you through creating your very first Astro project in whatever directory you run it in.
+
+```shell
+# npm
+npm init astro
+
+# yarn
+yarn create astro
+
+# pnpm
+pnpm create astro
+```
+
+> ⚔️ Prefer to go it alone? Read our [manual setup](/en/guides/manual-setup-new) instructions instead.
```

Comment:
When these pages replace the existing versions and lose their "-new", this link should be updated.